src/rollbar.php

```diff
@@ -237,6 +238,35 @@ public function queueSize() {
     }
 
     /**
+     * Run the checkIgnore function and determine whether to send the Exception to the API or not.
+     *
+     * @param  bool  $isUncaught
+     * @param  mixed $caller_args
+     * @param  mixed $payload
+     * @return bool
+     */
+    protected function _shouldIgnore($isUncaught, $caller_args, $payload)
+    {
+        try {
+            if (is_callable($this->checkIgnore)
+                && call_user_func_array($this->checkIgnore, [$isUncaught,$caller_args,$payload])
```

Comment:
http://php.net/manual/en/function.func-get-args.php

I know that this is a pretty 'boring' function that's hopefully rarely changed, but could you use `func_get_args` here (and if not, why not?). It would make sure that no matter what we changed about the function arguments it would still work.